src/login_nopam.c

```diff
@@ -67,18 +67,15 @@
 #define TABLE	"/etc/login.access"
 #endif
 
-/* Delimiters for fields and for lists of users, ttys or hosts. */
-static char fs[] = ":";		/* field separator */
-static char sep[] = ", \t";	/* list-element separator */
```

Comment:
sep is used far enough apart that having a #define for it is worthwhile here, lest someone change one user without noticing the other needs updating.